packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/gux-form-field-radio.tsx

```diff
@@ -36,10 +36,16 @@ export class GuxFormFieldRadio {
   private disabled: boolean;
 
   @State()
-  private hasError: boolean = false;
+  private hasHelp: boolean = false;
 
   @State()
-  private hasHelp: boolean = false;
+  private hasError: boolean = false;
+
+  @Prop()
+  hasGroupError: boolean = false;
```

Comment:
Consider using `getClosestElement` to get a reference to `gux-form-field-radio-group-beta` if it is a parent. Then watching the parent somehow to look at its state and reference it in this component.
Look at `'gux-sort-control'`, `'gux-segmented-control-item'` and `'gux-option-group-beta'` for inspiration.

Reply:
I tried this implementation that you suggested. I think it works as expected for the `hasGroupDisabled` case. However, watching `hasGroupError` from the parent radio-group component is needed for updating the error state of children when the `group-error` span is added or removed.